src/containers/BooksForm.js

```diff
@@ -0,0 +1,66 @@
+import React from 'react';
+import PropTypes from 'prop-types';
+import { bindActionCreators } from 'redux';
+import { connect } from 'react-redux';
+import { books } from '../actions';
+import categories from '../categories';
+
+function BooksForm({ createBook }) {
+  const [formData, setFormData] = React.useState({ title: '', category: '' });
+
```

Comment:
Define constant with Books categories in your BooksForm to store the various book categories. ([screenshot from pathwright](https://gitlab.com/microverse/guides/projects/requirements_screenshots/raw/master/images/react_reducx/bookstore/milestone2/Form/CategoryConstants.png))

Reply:
This is resolved well